extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -109,8 +109,7 @@ public function getTopContributorsData() {
 	 * @return array
 	 */
 	public function getTopAdminsData() {
-		$allAdmins = $this->getAllAdmins();
-		$allAdminsDetails = $this->getContributorsDetails( $allAdmins );
+		$allAdminsDetails = $this->getContributorsDetails( $this->usersModel->getAllAdmins() );
```

Comment:
What do you think about caching Contributors Details instead of results of query?